loadgen/kitchen_sink_executor_test.go

```diff
@@ -695,7 +668,7 @@ func TestKitchenSink(t *testing.T) {
 				},
 				WorkflowInput: &WorkflowInput{
 					InitialActions: ListActionSet(
-						NewTimerAction(2000), // timer to keep workflow open long enough for client action
+						NewTimerAction(5000), // timer to keep workflow open long enough for client action
```

Comment:
Trying to mitigate flaky tests. Better solution might be to add `AllHandlersFinished` but I'm not quite sure how to.